.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,5 +10,6 @@
 - [ ] All: Added an entry to [CHANGELOG.md](/ethereum/execution-spec-tests/blob/main/docs/CHANGELOG.md).
 - [ ] All: Considered updating the online docs in the [./docs/](/ethereum/execution-spec-tests/blob/main/docs/) directory.
 - [ ] Tests: All converted JSON/YML tests from [ethereum/tests](/ethereum/tests) have been added to [converted-ethereum-tests.txt](/ethereum/execution-spec-tests/blob/main/converted-ethereum-tests.txt).
+- [ ] Tests: All converted JSON/YML tests from [ethereum/tests](/ethereum/tests) have been removed.
```

Comment:
I don't think this check belongs in this repo's PR template. I feel like we should not proactively remove things from `ethereum/tests` until PRs in this repo are merged.

Reply:
This is so not to forget to attach a pr with removal of the tests from tests repo